common/src/main/scala/workers/WebWorkerF.scala

```diff
@@ -45,6 +58,12 @@ object WebWorkerF {
       def postMessage(message: js.Any): F[Unit] =
         Sync[F].delay(worker.postMessage(message))
 
+      def postTransferrable(buffer: Int8Array): F[Unit] =
+        Sync[F].delay(worker.postMessage(buffer, js.Array(buffer.buffer: dom.Transferable)))
+
+      def postTransferrable[A: Pickler](a: A): F[Unit] =
+        postTransferrable(asTransferable(a))
```

Comment:
This seems like a good general default implementation. We can put it in the `trait WebWorkerF`.